include/zephyr/bluetooth/mesh/cdb.h

```diff
@@ -123,14 +124,20 @@ void bt_mesh_cdb_iv_update(uint32_t iv_index, bool iv_update);
  *  @param num_elem Number of elements that the node has.
  *  @param net_idx NetIdx that the node was provisioned to.
  *
- *  @return The new node or NULL if it cannot be allocated.
+ *  @return The new node or NULL if CDB has already allocated
+ *  :kconfig:option:`CONFIG_BT_MESH_CDB_NODE_COUNT` nodes, or reached the end of the unicast
+ *  address range, or if @c addr is non-zero and less than the lowest available address or collide
```

Comment:
Could you use the same line length (presumably 80) as the existing documentation in this file? It looks a bit odd that you’re suddenly splitting lines quite differently than what’s already here. Generally, even though the Zephyr requirement has changed for 80 to 100, longer lines should be reserved to code only when it makes it more readable compared to splitting before 80.